Essentials/src/com/earth2me/essentials/IUser.java

```diff
@@ -1,7 +1,8 @@
 package com.earth2me.essentials;
 
+import com.earth2me.essentials.api.IAsyncTeleport;
+import com.earth2me.essentials.api.ITeleport;
```

Comment:
Even if it seems redundant, changing this import from `net.ess3.api.ITeleport` might cause ABI breakage with plugins that built against the class when it had the old import.